covid19-etl/tests/test_data/test_cityofchicago_input.csv

```diff
@@ -1,11 +1,11 @@
+"2022-04-28T00:00:00.000",,"0",,"62","76","64","55","32","24","19","8","0","198","139","3","70","35","52","121","16","46","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0",,,,,,,,,,,,,,,,,,
 "2022-04-27T00:00:00.000","812","0",,"193","196","183","95","63","46","22","14","0","496","311","5","127","84","119","294","60","128","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0",,,,,,,,,,,,,,,,,,
 "2022-04-26T00:00:00.000","911","0",,"220","227","185","116","76","57","24","6","0","524","378","9","156","65","155","349","65","121","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0",,,,,,,,,,,,,,,,,,
 "2022-04-25T00:00:00.000","922","0",,"214","188","198","108","98","61","31","24","0","512","402","8","151","78","133","339","63","158","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0",,,,,,,,,,,,,,,,,,
 "2022-04-24T00:00:00.000","400","0","7","75","64","99","63","48","29","14","8","0","229","169","2","71","27","47","177","23","55","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","1","0","1","0","1","4","0","4","3","0","0","1","2","4","0","0"
-"2022-04-23T00:00:00.000","407","0","1","66","68","94","58","48","41","23","9","0","215","189","3","87","34","54","148","24","60","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","1","0","0","0","0","0","1","0","1","0","0","0","0","0"
+"2022-04-23T00:00:00","407","0","1","66","68","94","58","48","41","23","9","0","215","189","3","87","34","54","148","24","60","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","1","0","0","0","0","0","1","0","1","0","0","0","0","0"
 "2022-04-22T00:00:00.000","683","0","10","152","126","148","98","63","48","34","14","0","392","289","2","135","68","98","236","48","98","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","2","1","2","1","0","1","2","1","0","8","2","0","2","1","3","3","1","0"
 "2022-04-21T00:00:00.000","679","0","18","113","168","160","80","69","50","23","16","0","357","317","5","115","53","116","251","45","99","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","4","4","0","2","2","3","3","0","12","6","0","0","0","9","9","0","0"
-"2022-04-20T00:00:00.000","717","0","11","97","190","177","100","75","36","26","16","0","417","294","6","113","67","90","293","33","121","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","2","2","0","0","2","2","3","0","7","4","0","0","1","5","5","0","0"
+,"717","0","11","97","190","177","100","75","36","26","16","0","417","294","6","113","67","90","293","33","121","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","0","2","2","0","0","2","2","3","0","7","4","0","0","1","5","5","0","0"
```

Comment:
it doesn't error out it just skips that rows and don't submit that data. and if that's missing from original dataset than we can't do anything :| Should it be errored out?

Reply:
i'm not sure, i think it might be a problem, if the date is missing from the real dataset and we skip the row, and they fix their dataset later, we will probably never submit this row

is this something you have really seen in the real dataset?

Reply:
So I have seen missing data for hospitalization but not for the other columns.

Reply:
i would say the ETL should not accept this

Reply:
Now I get it. I would be adding changes for it in the new commit.